File: Practica_1/src/gestorAplicacion/noPersonas/Cafe.java
package gestorAplicacion.noPersonas;



public class Cafe extends Bebida{
	
	private static int inventarioCafe;
	private static int codigoCafe = 1000;
	private static final float PRECIOCAFE = 5000;


	public Cafe() {
		super(codigoCafe, PRECIOCAFE);
		this.codigo = codigoCafe;
		codigo++;
		this.precio = PRECIOCAFE;
		inventarioCafe ++;
		
	}
	
	public static int getInventarioCafe() {
		return inventarioCafe;
	}

	public static void setInventarioCafe(int inventarioCafe) {
		Cafe.inventarioCafe = inventarioCafe;
	}

	public String toString() {
		String activo = "\nCafe";
		String formatoFactura = String.format("%-25s",activo).replace(' ', '.');
		formatoFactura += this.precio;
		return formatoFactura;
	}
	
	public static void prepararCafes(int i) {
		while(i>0) {
			new Cafe();
			i--;
		}
	}
	
	
	
	
	
	
}

File: Practica_2/noPersonas/cafe.py
from noPersonas.bebida import Bebida

class Cafe(Bebida):
    _inventarioCafe = 9
    _codigoCafe = 1000
    _PRECIOCAFE = 5000
    
    def __init__(self):
        super().__init__(Cafe._codigoCafe, Cafe._PRECIOCAFE)
        self._codigo = Cafe._codigoCafe
        self._codigo += 1
        self._precio = Cafe._PRECIOCAFE
        Cafe._inventarioCafe += 1
    
    @classmethod
    def getInventarioCafe(cls):
        return cls._inventarioCafe
    
    @classmethod
    def setInventarioCafe(cls, inventarioCafe):
        cls._inventarioCafe = inventarioCafe
    
    def __str__(self):
        activo = '\nCafe'
        formatoFactura = ('%-25s')%activo
        formatoFactura = formatoFactura.replace(' ', '.')
        formatoFactura += str(self._precio)
        return formatoFactura    

    @staticmethod
    def prepararCafes(i):
        while i > 0:
            Cafe()
            i += -1
    